src/nuketesting/_cli/runner.py

```diff
@@ -0,0 +1,165 @@
+from __future__ import annotations
+
+import platform
+import subprocess
+import sys
+from pathlib import Path
+
+import pytest
+
+import nuketesting
+from nuketesting.datamodel.constants import RUN_TESTS_SCRIPT
+
+
+class RunnerException(Exception):  # noqa: N818
```

Comment:
😂 Damn I know this noqa. It's because you named it "Exception" and not "Error" .... Had it far too often. We can disable it globally if you want.